platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/components/state/DefaultStateManagementComponent.java

```diff
@@ -49,9 +48,9 @@ public class DefaultStateManagementComponent implements StateManagementComponent
     private final SignedStateHasher signedStateHasher;
 
     /**
-     * A logger for hash stream data
+     * A logger handler for hash stream data
      */
-    private final HashLogger hashLogger;
+    private final Predicate<ReservedSignedState> hashLogger;
```

Comment:
I don't think `hashLogger` is a great name for this - it isn't a logger. Maybe something like `offerToHashLogger` would be more suitable, along with a javadoc describing the precise role the predicate fills - specifically, an offer method that returns whether the task was accepted or not

(this comment also applies to the naming of the constructor input parameter)

Reply:
good catch.